src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+  const titleMessage = document.createElement('h2');
+  const textMessage = document.createElement('p');
+
+  message.classList.add(`notification`);
+  message.classList.add(type);
```

Comment:
You can add multiple classes to an element at once by passing them as separate arguments to the 'classList.add' method. This would allow you to make your code more concise.